src/app/modules/group/components/manager-permission-dialog/manager-permission-dialog.component.html

```diff
@@ -0,0 +1,94 @@
+<p-dialog
+    [(visible)]="visible"
+    [modal]="true"
+    [draggable]="false"
+    [closable]="false"
+    styleClass="alg-permissions-edit-dialog">
+
+  <p-header>
+    <div class="dialog-header">
+      <span class="dialog-handler">
+        <i class="fa fa-lock"></i>
+      </span>
+      <span class="dialog-title" i18n>
+        {{ group?.name }}: manager access given to {{ userCaption }}
+      </span>
+      <span class="dialog-close" (click)="onClose()">
+        <img src="assets/images/close.svg" alt="close">
+      </span>
+    </div>
+  </p-header>
+
+  <div class="dialog-container">
+    <div class="dialog-content">
+      <p i18n>
+        This panel lets you select what you allow the selected user or group of users to do on the current group and its descendants.
+      </p>
+
+      <alg-progress-section
+          i18n-title title="Management level"
+          icon="fa fa-users-cog"
+          [(value)]="managerValues.canManage"
+          [values]="managementLevelValues"
+          [collapsible]="false"
+          [collapsed]="false"
```

Comment:
`[collapsible]="false"` + `[collapsed]="false"` is redundant, I think it's safe to assume that when the section is not collapsible, we don't want it to be collapsed

I spotted this in other parts of the PR, like for `<alg-boolean-section>`

Reply:
Since this PR is quite big already,  this could be done in another refactoring PR, what do you think ?

Reply:
Yes please, let's move it to separated ticket